testing/kubernetes/testing/fixture.go

```diff
@@ -0,0 +1,393 @@
+// Modified from the original at k8s.io/client-go/testing in line with
```

Comment:
Will we replace this module with `k8s.io/client-go/testing.go` in the future?

Reply:
No, my PR was rejected.  It's possible I can do it with more of a wrapper than a replacement, but we need something.